test/Select.test.tsx

```diff
@@ -46,7 +48,13 @@ describe('Select', () => {
 					validate={validate}
 					render={({ handleSubmit }) => (
 						<form onSubmit={handleSubmit} noValidate data-testid="form">
-							<Select label="Test" required={true} name="best" data={data} />
+							<Select
+								label={(label && 'Test') || undefined}
```

Comment:
Pretty sure you can just have `label={label && 'Test'}`

Reply:
Only now that #155 got merged in which uses ReactNode for label type. Before merge Typescript complained that boolean was being assigned to string. Pushed a new commit.

Reply:
Oh interesting! Thanks for noting that. =) I guess the prior was a good change then! =)